orchestration/dap_orchestration/resources.py

```diff
@@ -0,0 +1,135 @@
+import subprocess
+from typing import Any, List
+from dataclasses import dataclass
+
+from dagster import DagsterLogManager, Array, Field, resource, StringSource, IntSource
+from dagster.core.execution.context.init import InitResourceContext
+
+
+@dataclass
+class DataflowBeamRunner:
+    working_dir: str
+    logger: DagsterLogManager
+    # todo: **kwargs param to local beam runner + dataflow beam runner when we refactor this to use dagster_utils
+    region: str
+    worker_machine_type: str
+    autoscaling_algorithm: str
+    num_workers: int
+    max_num_workers: int
+    google_project: str
+
+    # note: Use "-> None" if function does not return a value
+    def __post_init__(self) -> None:
+        self.arg_dict = {
+            # resource config - how stuff behaves
+            # will not affect what is returned - will only affect run performance/success
```

Comment:
⛏️ I think the rest of the comments in this dict are not needed, but I'm 51/49 on that. Feel free to push back.

Reply:
+1